### PR TITLE
added: save song edit when enter key is pressed

### PR DESCRIPTION
**Pull request by Pekito**
**Describe the changes you made**
Add an onKeyDown event listener to the `edit.tsx` and `regex.tsx` component to check if the user pressed the Enter key and calling the saveEdit method.

**Additional context**
In the Issue #3774 @nikfiendluvr666 asked to the restore the ability to save the editted song when the enter key is pressed

### CODE REVIEW

File: src/ui/popup/edit.tsx [hunk 1]
```diff
@@ -133,7 +133,19 @@ export default function Edit(props: { tab: Resource<ManagerTab> }) {
 							/>
 							<Squircle id="coverArtClip" />
 						</a>
-						<div class={styles.songDetails}>
+						<div
+							class={styles.songDetails}
+							onKeyDown={(event) => {
+								if (event.key === 'Enter') {
```
Comment (yayuyokitano) [inline src/ui/popup/edit.tsx:139]:
```suggestion
								if (event.key === 'Enter' && !event.isComposing) {
```

File: src/ui/popup/regex.tsx [hunk 2]
```diff
@@ -62,7 +62,15 @@ export default function Regex(props: {
 	tab: Resource<ManagerTab>;
 }) {
 	return (
-		<div class={styles.regexContainer}>
+		<div
+			class={styles.regexContainer}
+			onKeyDown={(event) => {
+				console.log(event);
+				if (event.key === 'Enter') {
```
Comment (yayuyokitano) [inline src/ui/popup/regex.tsx:69]:
```suggestion
				if (event.key === 'Enter' && !event.isComposing) {
```

File: src/ui/popup/regex.tsx [hunk 3]
```diff
@@ -62,7 +62,15 @@ export default function Regex(props: {
 	tab: Resource<ManagerTab>;
 }) {
 	return (
-		<div class={styles.regexContainer}>
+		<div
+			class={styles.regexContainer}
+			onKeyDown={(event) => {
+				console.log(event);
```
Comment (yayuyokitano) [inline src/ui/popup/regex.tsx:68]:
```suggestion
```

File: src/ui/popup/edit.tsx (hunk 1 above)
Comment (yayuyokitano) [reply src/ui/popup/edit.tsx:139]: This causes errors with some IMEs for languages like chinese and japanese

File: src/ui/popup/regex.tsx (hunk 2 above)
Comment (yayuyokitano) [reply src/ui/popup/regex.tsx:69]: This causes errors with some IMEs for languages like chinese and japanese

**Review by yayuyokitano — CHANGES_REQUESTED** [review]
Thank you for your contribution! A couple of small changes needed before this can be pushed.

File: src/ui/popup/regex.tsx (hunk 2 above)
Comment (Pekito) [reply src/ui/popup/regex.tsx:69]: I didn't even know the existence of this property 😅. Going to commit the update in a moment.

File: src/ui/popup/edit.tsx (hunk 1 above)
Comment (Pekito) [reply src/ui/popup/edit.tsx]: if statement updated at https://github.com/web-scrobbler/web-scrobbler/pull/3785/commits/d5d993d8a34833f897bbcf2a61196ee5cd363ab1

File: src/ui/popup/regex.tsx (hunk 2 above)
Comment (Pekito) [reply src/ui/popup/regex.tsx]: changes made here -> https://github.com/web-scrobbler/web-scrobbler/pull/3785/commits/d5d993d8a34833f897bbcf2a61196ee5cd363ab1

File: src/ui/popup/regex.tsx (hunk 3 above)
Comment (Pekito) [reply src/ui/popup/regex.tsx]: removed this console.log here -> https://github.com/web-scrobbler/web-scrobbler/pull/3785/commits/d5d993d8a34833f897bbcf2a61196ee5cd363ab1

**Review by yayuyokitano — APPROVED** [review]
LGTM

**Comment by Pekito** [comment]
@yayuyokitano does this repo have a task roadmap? I would like to help with other changes in my free time

**Comment by yayuyokitano** [comment]
> @yayuyokitano does this repo have a task roadmap? I would like to help with other changes in my free time

We just did a major release last night, so right now we don't really have a roadmap, as we are currently mainly focusing on just getting bugs and issues fixed again and again until v3 becomes stable, and then going through and improving code quality.

If you want to help out, you can look out for bug reports and attempt to fix them, go through code and fix up code of low quality, you can add connectors suggested in issues (connector request issues are pretty much always labeled with connector and request labels), etc. We in general have more than enough issues to go around in the repository unfortunately, as prior to v3 there was a pretty long stagnant period.

I want to start building a roadmap for features as well soon-ish, but for now our priority is mainly on stability and getting rid of some tech debt until things start to feel solid.

We also have a [discord](https://discord.com/invite/u99wNWw) as well if you want more rapid and dynamic communication and bouncing of ideas

**Pull request merged**